src/components/InventoryTable/EntityTable.js

```diff
@@ -2,7 +2,6 @@ import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import { selectEntity, setSort } from '../../store/actions';
 import { useDispatch, useSelector } from 'react-redux';
-import { TableGridBreakpoint, TableVariant } from '@patternfly/react-table';
```

Comment:
There should be a least the `TableVariant` exported from the `/deprecated` module, maybe even the `TableGridBreakpoint`. If it is not found we may need to add an entry to the transforms.

Reply:
Seems wed have to go the transfromer route. The only possible exports from this are Table, TableBody, TableHeader

Reply:
Though I dont think this is causing any issues now. These are essentially just strings

Reply:
We should change these back and add the proper entries to the fec config.

Reply:
Actually. They don't need any entry `import { TableGridBreakpoint, TableVariant } from '@patternfly/react-table';` works just fine. 

I'm curious why these had to change in the first place? :)

Reply:
Reverting, but we had talked about this on stand up and decided that there shouldnt be two different instances for imports on the table. See as their strings though I see no problem with this